setup.py

```diff
@@ -15,7 +16,9 @@
     ],
     tests_require=[
         'mock>=1.0.1',
+        'flake8',
     ],
+    test_suite='nose.collector',
```

Comment:
Yeah nose is a nice and pretty widely used test runner. You can run it standalone too, like:

```
$ nosetests /path/to/a/python/package
```

and it will walk the Python package looking for classes and methods that appear to be tests. Additionally, nose is capable of making coverage reports, and machine readable test reports (which is handy if you want to have a CI system display nice looking data about how the tests went).

This particular change makes ```python setup.py test``` work, which is a common pattern in Python projects.